src/skin/frontend/base/default/omise/gateway/js/testmode.js

```diff
@@ -0,0 +1,8 @@
+window.addEventListener('load', function() {
+	var warning = document.createElement('div'); 
+	warning.innerHTML = Translator.translate('OMISE TEST MODE') + ' - \
+		<a target="_blank" href="https://www.omise.co/what-are-test-keys-and-live-keys">\
+		[ ' + Translator.translate('more info') +' ]</a>';
+	warning.className = 'omise-test-warning';
+	document.body.appendChild(warning);
+});
```

Comment:
small thing.. we could add break line here.

Reply:
@jonrandy you are rewritting html with no respect to any module from Magento that is designed for that. are you sure those HTML/CSS classes will exists in every themes?